signin.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Panda_login</title>
+  <link rel="icon" href="images/logo/favicon.ico">
+  <link rel="stylesheet" href="style.css">
```

Comment:
❗️ 수정요청
존재하지 않는 스타일파일이 연결되어 있네요~
아래처럼 수정해주시고 해당 파일 리뷰는 2주차 PR 참고해주세요!

```suggestion
  <link rel="stylesheet" href="auth.css">
```